tests/test_postgres_logical_replication.py

```diff
@@ -555,10 +637,137 @@ def test_run(self):
 
         lsn_3 = bookmark['lsn']
         self.assertTrue(lsn_3 >= lsn_2)
+        #----------------------------------------------------------------------
+        # invoke the sync job again after deleting a record using the 'id IN (SELECT ...)' format
+        #----------------------------------------------------------------------
+        print("delete row from source db")
+        with db_utils.get_test_connection('dev') as conn:
+            with conn.cursor(cursor_factory=psycopg2.extras.DictCursor) as cur:
+                cur.execute("DELETE FROM {} WHERE id IN (SELECT id FROM {} WHERE id=2)".format(canonicalized_table_name(test_schema_name, test_table_name, cur), canonicalized_table_name(test_schema_name, test_table_name, cur)))
```

Comment:
For future reference, this could be
``` python3
cur.execute("DELETE FROM {0} WHERE id IN (SELECT id FROM {0} WHERE id=2)".format(canonicalized_table_name(test_schema_name, test_table_name, cur))
```